app/styleguides/_default/components/clay-twitter.css

```diff
@@ -0,0 +1,48 @@
+.twitter {
+    display: block;
+    margin: 0 0 24px;
+    width: 100%;
+
+    @media screen and (min-width: 768px) {
+      margin: 0 0 40px;
+
+      & iframe,
+      & .twitter-tweet {
+        width: 460px !important; /* stylelint-disable-line declaration-no-important */ /* twitter tries to force the width at the element level, need the important to override */
+      }
+    }
+  }
+
+  .clay-tweet blockquote.twitter-tweet {
+    border-color: #eee #ddd #bbb;
+    border-radius: 5px;
+    border-style: solid;
+    border-width: 1px;
+    box-shadow: 0 1px 3px rgba(0, 0, 0, .15);
+    display: inline-block;
+    font-family: 'Helvetica Neue', Roboto, 'Segoe UI', Calibri, sans-serif;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 16px;
+    margin: 10px 5px;
+    max-width: 468px;
+    padding: 0 16px 16px;
+  }
+
+  .clay-tweet blockquote.twitter-tweet p {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 20px;
+  }
+
+  .clay-tweet blockquote.twitter-tweet a {
+    color: #1782a9;
+    font-weight: 400;
+    outline: 0;
+    text-decoration: none;
+  }
+
+  .clay-tweet blockquote.twitter-tweet a:focus,
+  .clay-tweet blockquote.twitter-tweet a:hover {
+    text-decoration: underline;
+  }
```

Comment:
This file seems like a duplicate from `/components/tweet`. Please see here https://github.com/clay/clay-starter/pull/16/files#diff-b1d44bb7432125c7c44b3922c58d3126R1